R/renv.R

```diff
@@ -0,0 +1,6 @@
+renv <- new.env()
```

Comment:
For my own learning at some point, this did _not_ work as `rlang::new_environment()`. If I tried to use that here, I would errors like this when building the package, i.e. it couldn't load the code into the environment:

```
Error in renv_abi_check <- function(packages = NULL, ..., libpaths = NULL,  : 
  could not find function "<-"
```

Reply:
I'd suggest putting this in `zzz.R` to be closer to where you actually use it.